Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.serialization.in

```diff
@@ -109,6 +109,10 @@ enum class WebCore::ApplePaySessionPaymentRequestShippingType : uint8_t {
     ServicePickup,
 };
 
+class WebCore::PaymentSessionError {
+    [SecureCodingAllowed=[NSError.class], Validator='platformError'] RetainPtr<NSError> platformError();
```

Comment:
This validator is not equivalent to what was there before.

Reply:
Yes, Validator='platformError' is redundant.